hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/shell/bucket/CreateBucketHandler.java

```diff
@@ -60,6 +66,17 @@
       defaultValue = "LEGACY")
   private AllowedBucketLayouts allowedBucketLayout;
 
+  @Option(names = {"--replication", "-r"},
+      description = "Replication value. Example: 3 (for Ratis type) or 1 ( for"
+          + " standalone type). Inthe case of EC type, the following valid"
```

Comment:
Minor typo "Inthe" -> "in the".

Also, I wonder we should avoid being so specific about the EC types allowed, as that may change over time, and then we will forget to change this help text. Perhaps say "In the case of EC, pass DATA-PARITY, eg 3-2, 6-3, 10-4".

Reply:
Thanks for the review again. It make sense, I will fix the description shortly.